index.html

```diff
@@ -0,0 +1,302 @@
+        </aside>
+      </dd>
+      <dt>
+        <dfn data-local-lt="digital identities">Digital identity</dfn>
+      </dt>
+      <dd>
+        <p>
+          A digital representation of an [=identity credential=], such as a
+          digital driver's license or passport, embodying verifiable claims
+          about an individual's identity. Issued by a trusted [=digital
+          identity/issuer=], it enables authenticated interactions.
+        </p>
+        <aside class="note">
+          [=Digital identities=] are represented as instances of the
+          {{DigitalIdentity}} interface.
+        </aside>
+      </dd>
+      <dt>
+        <dfn>Identity credential provider</dfn>
+      </dt>
+      <dd>
+        An application or service that provides a user interface for selecting
+        and/or querying a [=digital identity=], such as a digital wallet that
+        manages various identity documents and credentials.
+      </dd>
+      <dt>
+        <dfn data-for="digital identity">Request protocol</dfn>
+      </dt>
+      <dd>
+        A standardized format for requesting a [=digital identity=], designed
+        to ensure the secure, private, and interoperable exchange of identity
+        information. See section [[[#protocol-registry]]].
+      </dd>
+      <dt>
+        <dfn data-for="Digital identity">Issuer</dfn>
+      </dt>
+      <dd>
+        The entity that issues a [=digital identity=], such as a government
+        agency or certified organizations.
+      </dd>
+    </dl>
+    <h2>
+      Scope
+    </h2>
+    <p>
+      The following items are within the scope of this specification:
+    </p>
+    <ul>
+      <li>Requesting a [=digital identity=], including mechanisms for secure
+      presentation.
+      </li>
+    </ul>
+    <p>
+      The following items are out of scope:
+    </p>
+    <ul>
+      <li>Enrollment process for establishing a [=digital identity=].
+      </li>
+      <li>UI/UX considerations, with the exception of privacy considerations,
+      which are addressed to ensure the protection of user data during the
+      request process.
+      </li>
+    </ul>
+    <h2>
+      Extensions to the `Navigator` interface
+    </h2>
+    <pre class="idl">
+    partial interface Navigator {
+      [SecureContext, SameObject] readonly attribute CredentialsContainer identity;
+    };
+    </pre>
+    <h3>
+      The `identity` attribute
+    </h3>
+    <p data-dfn-for="Navigator">
+      The <dfn>identity</dfn> attribute provides access to the the underlying
+      {{CredentialsContainer}} for managing [=identity credentials=].
+    </p>
+    <h3>
+      Extensions to Credential Management API
+    </h3>
+    <aside class="issue" data-number="65"></aside>
+    <h2>
+      Extensions to `CredentialRequestOptions` dictionary
+    </h2>
+    <pre class="idl">
+    partial dictionary CredentialRequestOptions {
+      sequence&lt;IdentityRequestProvider&gt; providers;
```

Comment:
So, something that i think is a bit inconsistent with the CredentailsContainer but I think is actually good, is that the CredMan spec goes over in the [Credential Registry](https://w3c.github.io/webappsec-credential-management/#sctn-cred-type-registry) the [options member identifier](https://w3c.github.io/webappsec-credential-management/#credential-type-registry-options-member-identifier), which is an WebIDL attribute that tells you which credential type to use.

```js
await navigator.credentials.get({
  publicKey: /** give me a passkey */,
  otp: /** give me an SMS OTP */,
  identity: /** give me a federated assertion */,
});
```

Whereas `providers` here, would be for `DigitalIdentity`, but that seems like a general term that could apply to other `Identity` subclasses.

```js
await navigator.identity.get({
  providers: [] // what Identity interface do we expect to provide?
});
```

For example, if we introduced a `FederatedIdentity` in the future, then we wouldn't know whether "digital" identities or "federated" identities are being requested.

So, this is a bit different from the `navigator.credentials.get()` API, but I'm not sure if it is necessarily bad.

It is hard to entirely predict the future, but I think the biggest differentiation in the subclasses of `Identity` is actually going to come as a result of introducing new `protocols`. That is, I think that the only distinction between a `decentralized` identity and a `federated` identity is the mechanism that defines how to request them.

That is, if/when we'd like to expose FedCM into this API, we could introduce a new `protocol`, add it to the registry in this spec here, and continuing returning a `DigitalIdentity` or, if necessary, create a subclass of `Identity` that is a `FederatedIdentity`, and the way we would figure out which one to construct is based on `protocol`, NOT the [options member identifier](https://w3c.github.io/webappsec-credential-management/#credential-type-registry-options-member-identifier)  as used in `navigator.credentials.get()`.

I think this could work, but since it deviates from `navigator.credentials.get()` it seemed worth noting.

@bvandersloot-mozilla @marcoscaceres WDYT?

Reply:
We should maybe just note this issue and say that `providers` may change based on what you describe above.

Reply:
I'm going to resolve this here to unblock this draft, so that we can resolve asynchronously here in this issue:

https://github.com/WICG/digital-credentials/issues/65#issuecomment-1924941840